### ISSUE TITLE
Can't add MySQL database support for PHP 7.0

### ISSUE THREAD

**Issue by geerlingguy** [oerdnj/deb.sury.org#106]
Using your excellent PHP 7.0 repository, I can install PHP and related packages just fine, and at least back in the alpha and early beta stages, I remember there being available a `php7.0-mysqlnd` package to add MySQL support to PHP.

How can I add MySQL support to PHP now? I can't seem to get it working with the latest PHP release in the php-7.0 repository :(

**Comment by oerdnj** [comment]
Try just php-mysql, I dropped support for external MySQL library for now (and maybe forever). 

O. 

On August 21, 2015 9:43:52 PM GMT+02:00, Jeff Geerling <notifications@github.com> wrote:
>Using your excellent PHP 7.0 repository, I can install PHP and related
>packages just fine, and at least back in the alpha and early beta
>stages, I remember there being available a `php7.0-mysqlnd` package to
>add MySQL support to PHP.
>
>How can I add MySQL support to PHP now? I can't seem to get it working
>with the latest PHP release in the php-7.0 repository :(
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/oerdnj/deb.sury.org/issues/106

-- 
Ondřej Surý

**Comment by geerlingguy** [comment]
@oerdnj - If I add just that package, then it's installed with a PHP 5.5.x (default trusty system package) version, and doesn't seem to be accessible from the PHP 7.0 installation... is there a simple way to link the older module to PHP 7?

**Comment by oerdnj** [comment]
Try only php-mysql without any version. To confirm you can do apt-cache policy php-mysql

On August 21, 2015 9:50:48 PM GMT+02:00, Jeff Geerling <notifications@github.com> wrote:
>@oerdnj - If I add just that package, then it's installed with a PHP
>5.5.x (default trusty system package) version, and doesn't seem to be
>accessible from the PHP 7.0 installation... is there a simple way to
>link the older module to PHP 7?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/oerdnj/deb.sury.org/issues/106#issuecomment-133543061

-- 
Ondřej Surý

**Comment by geerlingguy** [comment]
Will do; I'll report back the outcome in a few minutes when the VM provisions. Thanks!

**Comment by geerlingguy** [comment]
@oerdnj - Thanks, it looks like it may have been an issue with `php-mysql` package installation when I had the `php7.0-fpm` installed—I was getting an error when the fpm package was doing its autoconfig—something about `www-data` user not being present.

However, in this _particular_ instance, I'm running PHP under Apache, so I don't need FPM. I've removed the set of installed PHP packages, redone all install steps, and now it seems like it might be working correctly.

I'm reprovisioning from scratch to see if that's the only issue, but it looks like the main issue here has been resolved, thanks!
